SPECS/golang/golang-1.22.spec

```diff
@@ -129,22 +128,12 @@ mkdir -p %{buildroot}%{gopath}/src/code.google.com/p/
 # This file is not necessary: recent Go toolsets have good defaults.
 # Keep the file, but leave it blank. This makes the upgrade path very simple.
 install -vdm755 %{buildroot}%{_sysconfdir}/profile.d
-cat >> %{buildroot}%{_sysconfdir}/profile.d/go-exports.sh <<- "EOF"
```

Comment:
I'm not sure we should remove `go-exports.sh` -- The comments above indicate that it's good to have a blank version there. So I think we should leave it.

Reply:
But to be clear, still remove the `rm .../go-exports.sh` in the `%postun` scriptlet -- it's unnecessary because `go-exports.sh` will be in the `BUILDROOT` and `rpm` itself will delete it.